barricade/src/main/res/drawable/ic_reset.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@mustafa01ali Do we want to make it mandatory for the user of this library to have vector support or should we use image drawables? Currently we have 2 vector drawables - red tick mark, and timer icon.